app/main/posts/modify/post-editor.directive.js

```diff
@@ -118,6 +121,14 @@ function PostEditorController(
 
             // Initialize values on post (helps avoid madness in the template)
             attributes.map(function (attr) {
+                // Create associated media entity
+                if (attr.input === 'upload') {
+                    var media = {};
+                    if ($scope.post.values[attr.key]) {
+                        media = $scope.post.values[attr.key][0];
+                    }
+                    $scope.medias[attr.key] = media;
```

Comment:
For consistency, could we just populate it with an empty object and leave that to be populated by the media directive? as far as I can see the id is never used from here